### ISSUE TITLE
pyinstaller (auto-py-to-exe) error: Project output will not be moved to output folder!

### ISSUE THREAD

**Issue by snmnima** [pyinstaller/pyinstaller#5024]
```
python v3.6.8
PyInstaller v3.6
auto-py-to-exe v2.7.5
os: windows 7
```
---
Command:

_pyinstaller --noconfirm --onedir --console --name "MyMainapp" --key "qXx53d02m2PrswB1" --debug "all"  "<Script Location>"_

---
When I want to convert a python file to exe (using auto-py-to-exe), these errors are shown to me:
```
An error occurred while packaging
Traceback (most recent call last):
  File "c:\users\nima\appdata\local\programs\python\python36\lib\site-packages\auto_py_to_exe\packaging.py", line 131, in package
    run_pyinstaller()
  File "c:\users\nima\appdata\local\programs\python\python36\lib\site-packages\PyInstaller\__main__.py", line 114, in run
    run_build(pyi_config, spec_file, **vars(args))
  File "c:\users\nima\appdata\local\programs\python\python36\lib\site-packages\PyInstaller\__main__.py", line 65, in run_build
    PyInstaller.building.build_main.main(pyi_config, spec_file, **kwargs)
  File "c:\users\nima\appdata\local\programs\python\python36\lib\site-packages\PyInstaller\building\build_main.py", line 734, in main
    build(specfile, kw.get('distpath'), kw.get('workpath'), kw.get('clean_build'))
  File "c:\users\nima\appdata\local\programs\python\python36\lib\site-packages\PyInstaller\building\build_main.py", line 681, in build
    exec(code, spec_namespace)
  File "C:\Users\Nima\AppData\Local\Temp\tmp8t9th72t\MyMainapp.spec", line 17, in <module>
    noarchive=True)
  File "c:\users\nima\appdata\local\programs\python\python36\lib\site-packages\PyInstaller\building\build_main.py", line 244, in __init__
    self.__postinit__()
  File "c:\users\nima\appdata\local\programs\python\python36\lib\site-packages\PyInstaller\building\datastruct.py", line 160, in __postinit__
    self.assemble()
  File "c:\users\nima\appdata\local\programs\python\python36\lib\site-packages\PyInstaller\building\build_main.py", line 500, in assemble
    for name, path, typecode in compile_py_files(new_toc, CONF['workpath']):
  File "c:\users\nima\appdata\local\programs\python\python36\lib\site-packages\PyInstaller\utils\misc.py", line 152, in compile_py_files


      with open(obj_fnm, 'rb') as fh:
    FileNotFoundError: [Errno 2] No such file or directory: '-'



Project output will not be moved to output folder

Complete.
```
if someone knows something, please help me! thank you.

**Comment by Legorooj** [comment]
This bug has been fixed in the development version. Please either `pip install -U https://github.com/pyinstaller/pyinstaller/archive/develop.zip`, or wait for the next release of PyInstaller - version 4.0 and any day now; we're waiting on our project lead to actually do the release.

**Comment by Salz0** [comment]
Your solution didn't help at all

**Comment by Legorooj** [comment]
If you're still experiencing this problem, please open a new issue @Salz0.
